src/pages/workshops/WorkshopView.js

```diff
@@ -1,82 +1,118 @@
 import React, { useEffect, useState } from "react";
 import "./Workshops.css";
 import Select from "react-select";
+import { Link } from "react-router-dom";
+import { WorkshopCreateForm } from "./CreateWorkshop";
 export const WorkshopsList = () => {
+    const [showPopup, setShowPopup] = useState(false);
     const [workshops, setWorkshops] = useState([]);
     const [sortBy, setSortBy] = useState("alphabetical");
     const [filteredWorkshops, setFilteredWorkshops] = useState([]);
     const [search, setSearch] = useState("");
     const [filter, setFilter] = useState("all");
-
+    const [hostOptions, setHostOptions] = useState([]);
+    const [reload, setReload] = useState(0);
```

Comment:
What is this state for? If it's just for causing a component re-render, try using a different method (maybe useEffect hooks?)